src/app/view/perspectives/post-it/post-it-perspective.component.ts

```diff
@@ -272,15 +273,16 @@ export class PostItPerspectiveComponent implements OnInit, OnDestroy {
     postIt.document = documentModel;
     postIt.initialized = Boolean(documentModel.id);
 
-    if (!postIt.initialized) {
-      postIt.order = -this.postIts.length;
-    } else {
-      postIt.order = this.postIts.length;
-    }
+    const direction = this.wasCreatedPreviously(postIt) ? 1 : -1;
+    postIt.order = this.postIts.length * direction;
```

Comment:
What is this calculation all about?

Reply:
Until we have post-it orders stored. I've just assigned each post-it it's index as it's order number.

Reply:
Just a temporary, until post-it order is stored.
Direction makes sure, that post-its created in current session get moved to the front, while others are kept in their original order.